readme.md

```diff
@@ -363,6 +363,8 @@ This list is ordered based on the name without prefix, so excluding
     currently supporting [ESLint](https://github.com/Qard/remark-lint-code-eslint)
 *   [`remark-lint-match-punctuation`](https://github.com/laysent/remark-lint-plugins/tree/master/packages/remark-lint-match-punctuation)
     — Ensures punctuations are used in pairs if necessary.
+*   [`remark-lint-mdash-style`](https://github.com/alexandrtovmach/remark-lint-mdash-style)
+    — Ensure a mdash style to follow a standard format
```

Comment:
```suggestion
    — Ensure mdash style follows a standard format
```

`mdash`  is pronounced em-dash, so it would be `an` not `a` (annoying English!). I suggest making it shorter as well, what do you think?